model/src/cfd.rs

```diff
@@ -129,13 +129,23 @@ pub struct Order {
     pub id: OrderId,
 
     pub trading_pair: TradingPair,
+
+    /// The maker's position
+    ///
+    /// Since the maker is the creator of this order this always reflects the maker's position.
+    /// When we create a Cfd we change it to be the position as seen by each party, i.e. flip the
+    /// position for the taker.
```

Comment:
I appreciate the documentation but I still think using `serde rename` would help here (as I commented elsewhere) and below.